plugins/teststeps/cmd/cmd.go

```diff
@@ -28,8 +29,24 @@ var Name = "Cmd"
 
 var log = logging.GetLogger("teststeps/" + strings.ToLower(Name))
 
+// event names for this plugin.
+const (
+	EventCmdStart = event.Name("CmdStart")
```

Comment:
in order to further test this PR's behaviour, I needed valid step events, and noticed that the Cmd plugin declared but did not emit start and end events, so I implemented them here